src/templates/cv_template.html

```diff
@@ -226,72 +261,514 @@
+         <div class="heading_10">Digital Media</div>
+         <div class="regular_10">
+          <% _.forEach(digitalMedias, function(item) { %>
+            <p><%= item['artisticWork'] %></p>
+          <% }); %>
+         </div>
+       </div>
+      <% } %>
+     
+      <% if (drawings.length > 0) { %>
+        <div class="row">
+         <div class="heading_10">Drawings</div>
+         <div class="regular_10">
+          <% _.forEach(drawings, function(item) { %>
+            <p><%= item['artisticWork'] %></p>
+          <% }); %>
+         </div>
+       </div>
+      <% } %>
+
+      <% if (exhibits.length > 0) { %>
+        <div class="row">
+         <div class="heading_10">Exhibits</div>
+         <div class="regular_10">
+          <% _.forEach(exhibits, function(item) { %>
+            <p><%= item['artisticWork'] %></p>
+          <% }); %>
+         </div>
+       </div>
+      <% } %>
+
+      <% if (films.length > 0) { %>
+        <div class="row">
+         <div class="heading_10">Films</div>
+         <div class="regular_10">
+          <% _.forEach(films, function(item) { %>
+            <p><%= item['artisticWork'] %></p>
+          <% }); %>
+         </div>
+       </div>
+      <% } %>
+
+      <% if (graphicDesigns.length > 0) { %>
+        <div class="row">
+         <div class="heading_10">Graphic Designs</div>
+         <div class="regular_10">
+          <% _.forEach(graphicDesigns, function(item) { %>
+            <p><%= item['artisticWork'] %></p>
+          <% }); %>
+         </div>
+       </div>
+      <% } %>
+
+      <% if (illustrations.length > 0) { %>
+        <div class="row">
+         <div class="heading_10">Illustrations</div>
+         <div class="regular_10">
+          <% _.forEach(illustrations, function(item) { %>
+            <p><%= item['artisticWork'] %></p>
+          <% }); %>
+         </div>
+        </div>
+      <% } %>
+
+      <% if (installations.length > 0) { %>
+        <div class="row">
+         <div class="heading_10">Installations</div>
+         <div class="regular_10">
+          <% _.forEach(installations, function(item) { %>
+            <p><%= item['artisticWork'] %></p>
+          <% }); %>
+         </div>
+       </div>
+      <% } %>
+
+       <% if (lightingDesigns.length > 0) { %>
+       <div class="row">
+         <div class="heading_10">Lighting Designs</div>
+         <div class="regular_10">
+          <% _.forEach(lightingDesigns, function(item) { %>
+            <p><%= item['artisticWork'] %></p>
+          <% }); %>
+         </div>
+       </div>
+      <% } %>
+
+       <% if (motionGraphicss.length > 0) { %>
+       <div class="row">
+         <div class="heading_10">Motion Graphics</div>
+         <div class="regular_10">
+          <% _.forEach(motionGraphicss, function(item) { %>
+            <p><%= item['artisticWork'] %></p>
+          <% }); %>
+         </div>
+       </div>
+      <% } %>
+
+       <% if (multipleTypess.length > 0) { %>
```

Comment:
maybe rename "multipleTypes" ? (see also line 464)

Reply:
So, these artisticWorks are populated automatically using transform... it pluralizes all kinds of artisticWorks... With the above one, the type itself has 's' at the end so it adds a extra one...might be a fair bit of rework...

Reply:
Ah, makes sense. I'll approve in that case.